GeodesyMLToSiteLog/GeodesyML/schemas/monumentInfo.xsd

```diff
@@ -144,76 +144,118 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
             <enumeration value="Scotia"/>
         </restriction>
     </simpleType>
-    <complexType name="formInformationType">
+
+    <complexType name="FormInformationPropertyType">
         <sequence>
-            <element name="preparedBy" type="string"/>
-            <element name="datePrepared" type="gml:TimePositionType" minOccurs="0"/>
-            <element name="reportType" type="string"/>
+            <element ref="geo:FormInformation" minOccurs="0"/>
         </sequence>
+        <attributeGroup ref="gml:AssociationAttributeGroup"/>
     </complexType>
-    <complexType name="moreInformationType">
+
+    <element name="FormInformation" type="geo:FormInformationType"/>
+
+    <complexType name="FormInformationType">
+        <complexContent>
+            <extension base="gml:AbstractGMLType">
+                <sequence>
+                    <element name="preparedBy" type="string"/>
+                    <element name="datePrepared" type="gml:TimePositionType"/>
```

Comment:
It's still unclear why datePrepared on left has minOccurs="0".

Reply:
There is still a question here. Can you try to work out why there is this difference in minOccurs?

Reply:
here

Reply:
I can add `minOccurs="0"` back to `datePrepared ` if you should keep it. I removed it because it was not in monumentInfo.xsd v0.5 in GeodesyML repository.

Reply:
I was wondering why was this added at some point in the past. If it was added for a good reason, then we don't want to remove it. But if it was added by mistake, then we do want to remove it. I'll will try work out why was this field made optional in the converter at some stage.